README.md

```diff
@@ -439,6 +452,9 @@ Opensearch Dashboards. Following services are available after start up:
 | Opensearch Dashboards: | `localhost:5601` |
 | Grafana Dashboards: | `localhost:3000` |
 | Prometheus: | `localhost:9090` |
+| Nginx: | `localhost:8081` |
+| Keycloak: | `localhost:8080` |
+| Postgres: | `localhost:5432` |
 
 The credentials for Grafana are `admin` and `admin`.
```

Comment:
Thanks for updating the readme. We are starting to get more services with credentials though. Maybe you could extend the table with another column and add the credentials for nginx (`user`, `password`) and keycloak (`admin`, `admin`), and postgres probably also.

Reply:
done